app/src/main/java/com/gh4a/adapter/TrendAdapter.java

```diff
@@ -57,6 +58,11 @@ public void onBindViewHolder(ViewHolder holder, Trend trend) {
         String desc = trend.getDescription();
         holder.tvDesc.setText(desc != null ? EmojiParser.parseToUnicode(desc) : null);
 
+        String lang = trend.getLanguage();
+        if (TextUtils.isEmpty(lang)) {
+            lang = "Unknown";
```

Comment:
This should use a string resource (use the `setText` variant that takes a string resource in that case).